code/modules/mob/living/silicon/ai/vox_sounds.dm

```diff
@@ -97,7 +94,6 @@ GLOBAL_LIST_INIT(vox_sounds, list("," = 'sound/vox_fem/,.ogg',
 "cable" = 'sound/vox_fem/cable.ogg',
 "call" = 'sound/vox_fem/call.ogg',
 "called" = 'sound/vox_fem/called.ogg',
-"canal" = 'sound/vox_fem/canal.ogg',

```

Comment:
Why

Reply:
Tell me what this'll be used for.

Reply:
To say centimeter as AI?

Reply:
Canal to say centimeter? Okay? How?

Also why would an AI need to say Centimeter.

Reply:
Nevermind, I'm retarded. For some reason I read "Canal" as "centi'